src/ts/ConvivaAnalyticsTracker.ts

```diff
@@ -1,16 +1,16 @@
 import * as Conviva from '@convivainc/conviva-js-coresdk';
-import type {
+import {
   AudioTrack,
   ErrorEvent,
   PlaybackEvent,
   PlayerAPI,
-  PlayerEvent,
   PlayerEventBase,
   SourceConfig,
   VideoQualityChangedEvent,
   SubtitleTrack,
   TimeMode,
 } from 'bitmovin-player';
+import { PlayerEvent } from 'bitmovin-player';
```

Comment:
Because of IDE autoimport, fixed here https://github.com/bitmovin/bitmovin-player-web-analytics-conviva/pull/115/commits/fda346eba962b2411e222bd48b70de119aa0c6ed.